programs/server/tests/README.md

```diff
@@ -0,0 +1,8 @@
+This folder contains a set of tests for HTML applications in 'server' module.
```

Comment:
Do we have plans to enable it on the CI?

Reply:
It would be great to have it run by CI, but I do not know how to run these tests only if there are changes in the programs/server folder, and not in any other part of the ClickHouse. So these tests won't slow down the main pipeline

I think we can live for some period with these tests run manually by PR author/reviewer in case if there are changes to dashboard.html/play.html files. 

Next, either I will learn CH testing infrastructure better, or someone else can help to enable the tests correctly. 
Until that the tests can be considered not as blockers, but as development helpers.

Reply:
Yes, we have the possibility to run some checks only if some subset of files is modified.

Reply:
Do you want to include this support into this PR? In this case I'll need a contact/link to docs to learn how to make it right.

Of we can leave it for some future PR and in this case I need instructions what to fix/improve in the current one.